src/SafeDNS/SettingsClient.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace UKFast\SDK\SafeDNS;
+
+use UKFast\SDK\SafeDNS\Entities\Settings;
+
+class SettingsClient extends Client
+{
+    const SETTINGS_MAP = [
+        'custom_soa_allowed' => 'customSoaAllowed',
+        'custom_base_ns_allowed' => 'customBaseNsAllowed',
+        'custom_axfr' => 'customAxfr',
+        'delegationAllowed'
```

Comment:
Is this missing a key?

```suggestion
        'delegation_allowed' => 'delegationAllowed'
```